CHANGES.txt

```diff
@@ -18,6 +18,9 @@ Breaking Changes
 Changes
 =======
 
+- Greatly improved performance when using a scalar function inside a ``WHERE``
+  clause.
```

Comment:
Greatly improved performance of queries that use scalar functions inside the ``WHERE`` clause.